tokens/custom-spectrum/custom-vars.css

```diff
@@ -36,9 +36,9 @@ governing permissions and limitations under the License.
   --spectrum-animation-ease-out: cubic-bezier(0, 0, 0.40, 1);
   --spectrum-animation-ease-linear: cubic-bezier(0, 0, 1, 1);
 
-   /* TODO: refactor to use a translation layer that would adjust current `corner-radius-1000` to be 9999px instead of 0.5 */
-  /* should be used in place of `--spectrum-corner-radius-1000` and -`-spectrum-corner-radius-full`*/
   --spectrum-corner-radius-1000-full: 9999px;
+  --spectrum-corner-radius-full: var(--spectrum-corner-radius-1000-full);
+  --spectrum-corner-radius-1000: var(--spectrum-corner-radius-1000-full);
```

Comment:
This overrides the current `--spectrum-corner-radius-1000` and `--spectrum-corner-radius-full` with the value we want, if there's a different way to do this that I overlooked, let me know!

Reply:
I think this is fine for now... Do you remember if Jenn said anything about where she left this? Had she started talking to Garth about getting this added to the tokens system?

Reply:
We chatted about this in a Slack DM with Patrick Fulton and determined that the current setup in this PR is ok for now 👍